src/Decimal.php

```diff
@@ -354,21 +354,19 @@ public function multiply($value, ?int $scale = null)
      * Divide this Decimal by $value and return the quotient as a new Decimal.
      *
      * @param string|int|float|static $value
-     * @param int|null $scale
+     * @param int $scale
      *
      * @throws \LogicException if $value is zero.
      *
      * @return static
      */
-    public function divide($value, ?int $scale = null)
+    public function divide($value, int $scale)
     {
         $decimal = static::create($value);
         if ($decimal->isZero()) {
             throw new LogicException('Cannot divide by zero. Only Chuck Norris can!');
         }
 
-        $scale = $this->resultScale($this, $decimal, $scale);
```

Comment:
Shouldn't that method be called `resolveScale`? @dereuromark

Reply:
We could, but that's probably a minor thing.